config/initializers/git_sha.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+revisions_logfile = Rails.root.join("..", "..", "revisions.log")
+
+GIT_SHA =
+  if File.exist?(revisions_logfile)
+    `tail -1 #{revisions_logfile}`.chomp.split(" ")[3].gsub(/\)$/, "")
```

Comment:
Perhaps a future improvement could be to cache this value

```
tail -1 #{revisions_logfile}`.chomp.split(" ")
```

so that we don't `tail` this file three times (which requires a call to the shell). 

This is probably not an issue for bibdata where this page is not loaded very often, but it would be an issue other projects, like Orangelight, where the page is displayed quite often.

Reply:
The values themselves (GIT_SHA, BRANCH, and LAST_DEPLOYED) are calculated only once, right? since they are in UPPERCASE.